docs/DMS/PRIMARY-DATA-STORAGE/RELATIONAL-QUERIES.md

```diff
@@ -103,14 +139,14 @@ CREATE TABLE [dbo].[QueryStudent] (
 );
 END
 
--- Example SQL query for a lastSurname. Join includes partition elimination on Documents table
+-- Example: Query by specific field (includes partition elimination)
 SELECT d.edfi_doc
-FROM Documents d INNER JOIN QueryStudent q
-  ON (d.partition_key = q.document_partition_key AND d.id = q.document_id)
+FROM Documents d
+INNER JOIN QueryStudent q ON (d.partition_key = q.document_partition_key AND d.id = q.document_id)
```

Comment:
Any reason not to simply call this `Student` instead of `QueryStudent`?

Reply:
The naming intent was to show that these are the optional Query tables.